dev/SwipeControl/SwipeControl.cpp

```diff
@@ -678,12 +678,21 @@ void SwipeControl::AttachDismissingHandlers()
         if (auto xamlRoot = uiElement10.XamlRoot())
         {
             auto xamlRootContent = xamlRoot.Content();
+            m_onXamlRootPointerPressedEventHandler = AddRoutedEventHandler<RoutedEventType::PointerPressed>(
```

Comment:
I'm pretty sure you need to do 
```
if (auto xamlRootContent = ...)
{
   ...
}
```

Reply:
I thought about this, I think the only way for xamlRoot.Content() to be null is during cleanup scenarios, which Attach* can't occur during. That being said it might just be safer to check.